### DISCUSSION TITLE
Shelly pro 3em / Shelly pro 2pm

### DISCUSSION THREAD

**Discussion by StefanSauensiek** [camueller/SmartApplianceEnabler#548] [Allgemein]
Hallo an alle,

ich habe meine S0 Stromzähler durch Shelly pro 3em und Shelly pro 2pm.
Das mit dem S0 hat super funktioniert, braucht aber zu viel Platz imSchaltschrank.
Als ich die Einstellungen im Smart Appliance Enabler anpassen wollte, musste ich feststellen, das da angehängten Einstellungen nicht zum Erfolg führen. In den Logs steht immer no data.
Hat jemand vielleicht die richtigen Einstellungen für mich?

Viele Grüße
Stefan

**Comment by camueller** [comment]
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Shelly_DE.md

**Reply by StefanSauensiek** [reply]
Moin Carsten und Danke für die schnelle Reaktion.
Ich habe jetzt alle möglichen Einstellungen die vorgeschlagen waren probiert, aber leider ohne Erfolg.
Die Eisntellungen für den Shelly pro 4pm habe ich für den Shelly pro 2pm benutzt und die Einstellungen für den EM habe ich für den pro 3em genutzt. Habe mal die beiden Logs angehängt.
Die Logdatei für den 2PM ist für Kanal 0.

Gruß
Stefan

![Log 2pm](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/aa1df7e3-6976-4a07-8885-d8d0c1fd87d5)
![Log 3em](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/c662064d-dc7a-47d4-802e-fa55bf827ab9)

**Comment by StefanSauensiek** [comment]
Hallo noch mal,

nach einigem Probieren und Abfragen des Shellystatus konnte ich für einen Shelly pro 2pm die Einstellungen finden. 

![SAE fuer Shelly pro 2pm](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/2590cb3c-2ebb-41db-9ef7-2b8a33ec1303)

Für Einstellungen für den Shelly pre 3em bekomme ich nicht hin. Habe im Augenblick keine Ahnung wo der Fehler liegt.
Einstellungen:
![SAE für Shelly pro 3em](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/366e8001-6566-44a8-9767-46f8f04358a7)

Log:
![Log pro 3em](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/105842e2-bd62-4dd3-bd84-3f99a3aa8de5)

Im Log kommt der Pfad wie in den Einstellungen vor, aber im Sunnyportal bleibt die Anzeige immer bei 0.

Bin um jeden Hinweis dankbar.

Stefan

**Comment by camueller** [comment]
In Deinem Screenshot wird ja die Response vom Shelly angezeigt, d.h. Du kannst den [JSON-Path bestimmen](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ValueExtraction_DE.md#mit-json-pfad) und dann konfigurieren.

**Reply by StefanSauensiek** [reply]
Lieber Carsten,
Leider bin ich was das angeht, total unbeleckt und lerne jede Minute etwas Neues, aber jetzt hast du mir Licht ans Fahrrad gemacht.
Vielen Dank für die Mühe und die Geduld. Super Service.
Ich habe die Einstellungen mal angehängt, falls es von Interesse ist.

Gruß
Stefan

![SAE Einstellungen für Shelly pro 3em](https://github.com/camueller/SmartApplianceEnabler/assets/121601981/41f8acd2-6c26-4d72-ab66-48fabe7ea04a)
